code/index.html

```diff
@@ -1,22 +1,82 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+<html lang="en">
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <title> Project 2: The News site</title>
+</head>
 
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+<body>
+
+  <header class="header-landing-page">
+    <div class="header-logo-container">
+      <img class="header-logo" src="./assets/Circle-Forset-Night-bw (1).png" alt="Site logo">
+    </div>   
+    <h1 class="site-title">HIKING</h1>
+    <div class="foot-step-right-side">      
+      <img class="foot-step" src="./assets/foot-steps.png" width="60px">
+      <img class="foot-step" src="./assets/foot-steps.png" width="60px">
+      <img class="foot-step" src="./assets/foot-steps.png" width="60px">      
+    </div>
+    <nav class="navigation-container">
+      <div class="navigation-item">
+        <a class="nav-link" href="https://google.com">About us</a>
+      </div>
```

Comment:
what is the div for? seems unnecessary to wrap a single item in a div like this. add the `navigation-item` class to the `<a>` tag instead and style it how you want it.